src/NServiceBus.Core/Serializers/SystemJson/SystemJsonSerializerSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
+namespace NServiceBus.Serializers.SystemJson
+{
+    using System.Text.Json;
+
+    class SystemJsonSerializerSettings
```

Comment:
Would it make sense to make this public? Let's say for example for DynamoDB we would want to implement a functionality that allows sharing the options of the main serializer with the saga serializer. Im such a case we would have to use reflection or dynamic typing to get the values out right?

Or call it YAGNI for now?

Reply:
The other option would be to let the user pass in this object from outside. But this might be leaving the land of familiar serialization config APIs and be too much in the new transport seam territory

Reply:
Easy to add in a minor, hard to take away.